schematools/utils.py

```diff
@@ -89,61 +92,98 @@ def def_from_url(base_url, data_type, dataset_name):
     return schema_lookup[dataset_name]
 
 
-def schema_def_from_file(filename) -> Dict[str, types.DatasetSchema]:
+def schema_def_from_file(filename: Union[Path, str]) -> Dict[str, DatasetSchema]:
     """Read schema definitions from a file on local drive."""
     with open(filename, "r") as file_handler:
         schema_info = json.load(file_handler)
-        return {schema_info["id"]: types.DatasetSchema.from_dict(schema_info)}
+        return {schema_info["id"]: DatasetSchema.from_dict(schema_info)}
 
 
-def profile_def_from_file(filename) -> Dict[str, types.DatasetSchema]:
+def profile_def_from_file(filename: Union[Path, str]) -> Dict[str, DatasetSchema]:
     """Read a profile from a file on local drive."""
     with open(filename, "r") as file_handler:
         schema_info = json.load(file_handler)
-        return {schema_info["name"]: types.DatasetSchema.from_dict(schema_info)}
+        return {schema_info["name"]: DatasetSchema.from_dict(schema_info)}
 
 
-def schema_fetch_url_file(schema_url_file):
+def schema_fetch_url_file(schema_url_file: str) -> Dict[str, Any]:
     """Return schemadata from URL or File"""
 
     if not schema_url_file.startswith("http"):
         with open(schema_url_file) as f:
-            schema_location = json.load(f)
+            schema_data = json.load(f)
     else:
         response = requests.get(schema_url_file)
         response.raise_for_status()
-        schema_location = response.json()
+        schema_data = response.json()
+
+    return cast(Dict[str, Any], schema_data)
+
+
+CAMEL_CASE_REPLACE_PAT: Final[Pattern[str]] = re.compile(
```

Comment:
_CAMEL_CASE_REPLACE_PAT? We don't seem to be working with __all__ in this module.